_data/sites.json

```diff
@@ -6381,6 +6392,16 @@
         ]
     },
 
+    {
+        "name": "Sketchfab",
+        "url": "https://sketchfab.com/settings/account",
+        "difficulty": "easy",
+        "notes": "You have to click on the button Delete your account, select the consent to delete all content and confirm.",
+        "domains": [
+            "sketchfab.com"
+        ]
+    },
+
```

Comment:
This has to be between "Simplenote" and "Skillshare", to preserve the alphanumeric order